presto-hive/src/main/java/com/facebook/presto/hive/orc/OrcBatchPageSource.java

```diff
@@ -115,7 +115,7 @@ public OrcBatchPageSource(
         ImmutableList.Builder<Type> typesBuilder = ImmutableList.builder();
         for (int columnIndex = 0; columnIndex < columns.size(); columnIndex++) {
             HiveColumnHandle column = columns.get(columnIndex);
-            checkState(column.getColumnType() == REGULAR, "column type must be regular");
+            checkState(column.getColumnType() == REGULAR, format("column type of %s must be regular", column.getName()));
```

Comment:
this is nice. Would be handy to also add what the actual column type was to the error message.